scripts/capa2yara.py

```diff
@@ -180,7 +180,8 @@ def do_statement(s_type, kid):
             logger.info("doing api: " + repr(api))
 
             #    e.g. kernel32.CreateNamedPipe => look for kernel32.dll and CreateNamedPipe
-            if "." in api:
+            # TODO: properly handle .NET API calls
+            if api.count(".") == 1:
```

Comment:
maybe use `"::" in api` to detect .NET and handle those separately.

this check won't work very well as-is due to examples like:

```
  insn: token(0x6000033)+0x1: api(System.Object::.ctor)
```

Reply:
maybe slightly better now